src/main/kotlin/com/chattriggers/ctjs/minecraft/libs/renderer/Image.kt

```diff
@@ -49,6 +49,15 @@ class Image constructor(var image: BufferedImage?) {
         return texture
     }
 
+    /**
+     * Clears the image from GPU memory and removes its references CT side
+     * that way it can be garbage collected if not referenced in js code.
+     */
+    fun destroy(){
+        texture.deleteGlTexture()
+        CTJS.images.remove(this)
```

Comment:
Since we are now iterating this list, it would be best to make `images` a set, since we don't need ordering.